terraform/etl/49-lambda-streetscene-ingestion.tf

```
@@ -120,3 +121,46 @@ resource "aws_iam_role_policy_attachment" "streetscene_street_systems_lambda_sec
   role       = aws_iam_role.streetscene_street_systems_ingestion[0].name
   policy_arn = aws_iam_policy.streetscene_street_systems_lambda_secret_access[0].arn
 }
+
+module "street-systems-api-ingestion" {
+  count                          = !local.is_production_environment ? 1 : 0
+  source                         = "../modules/aws-lambda"
+  lambda_name                    = "street-systems-export"
+  identifier_prefix              = local.short_identifier_prefix
+  tags                           = module.tags.values
+  handler                        = "main.lambda_handler"
+  lambda_artefact_storage_bucket = module.lambda_artefact_storage_data_source.bucket_id
+  s3_key                         = "street-systems-api-ingestion.zip"
+  lambda_source_dir              = "../../lambdas/street-systems-api-ingestion"
+  lambda_output_path             = "../../lambdas/street-systems-api-ingestion.zip"
+  environment_variables          = {
+    API_SECRET_NAME       = "/data-and-insight/streets_systems_api_key"
+    OUTPUT_S3_FOLDER      = "${module.landing_zone_data_source.bucket_arn}/streetscene/*"
+    TARGET_S3_BUCKET_NAME = "streetscene/traffic-counters/"
+    API_URL = "https://flask-customer-api.ki8kabg62o4fg.eu-west-2.cs.amazonlightsail.com"
+  }
+  layers = [
+    "arn:aws:lambda:eu-west-2:${data.aws_caller_identity.data_platform.account_id}:layer:requests-2-31-0-and-httplib-0-22-0-layer:1",
+    "arn:aws:lambda:eu-west-2:${data.aws_caller_identity.data_platform.account_id}:layer:AWSSDKPandas-Python39:12",
```

This one was OK, 336392948345 is where the AWS managed layers live and is the same regardless of where you're deploying to - sorry my fault
```suggestion
    "arn:aws:lambda:eu-west-2:336392948345:layer:AWSSDKPandas-Python39:12",
```